### ISSUE TITLE
How to calculate "last frame average" in TABLE Ⅲ

### ISSUE THREAD

**Issue by LyTingHub** [intelligolabs/HARPER#2]
Dear author, thanks for your great work!

I want to know how to calculate "last frame average" in TABLE Ⅲ.  In the paper, it is described as "the average over the last frame of each action instance (Last frame average).", but I don't really understand. Does it 
Could you explain it for me or offer the code about evaluation?

Besides, I want to know how to get the data used for HPF benchmark. 
I failed to get the same number of sequence described in paper that "For 3D-HPF, we sampled 7917 sequences (of 20 frames each) for the training set and 3088 for the test set". 

Thanks for your help!

**Comment by andreaavo9** [comment]
In order to compute the last frame average we compute the MPJPE between the last frame of the GT sequence and the last frame of the predicted sequence.

The code that we are using to compute MPJPE is:
`import torch

def mpjpe_error(batch_pred,batch_gt): 
    batch_pred=batch_pred.contiguous().view(-1,3)
    batch_gt=batch_gt.contiguous().view(-1,3)

    return torch.mean(torch.norm(batch_gt-batch_pred,2,1))`

**Comment by LyTingHub** [comment]
Thanks for your prompt reply !
